src/index.js

```diff
@@ -1,2 +1,57 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render({ count }) {
+  function countUp() {
+    return render({ count: count + 1 });
+  }
+
+  function setCount(value) {
+    return render({ count: value });
+  }
+
+  const element = (
+    <div id="hello" className="greeting">
+      <p>Hello, world!</p>
+      <p>
+        <button type="button" onClick={() => countUp()}>
```

Comment:
```suggestion
        <button type="button" onClick={countUp}>
```

이렇게 함수를 넘겨주는 형태로도 사용이 가능해요.
즉시 실행하는 함수 형태와는 어떤 차이점이 있을까요?

Reply:
arrow function을 쓰면 렌더할 때마다 새로운 함수가 생성되지만 그냥 countUp을 쓰면 그런 일을 막을 수 있어서 더 좋네요! 값을 넘겨줄 필요가 없으면 arrow function을 안 쓰도록 하겠습니다. 리뷰 감사합니다!